contrib/cirrus/setup_environment.sh

```diff
@@ -483,6 +483,15 @@ then
       "--disablerepo=fedora*" "--disablerepo=updates*"
 fi
 
+# Fedora script command install
+case "$DISTRO_NV" in
+    fedora-42|fedora-42-aarch64|rawhide)
+        showrun dnf install -y util-linux-script
+        ;;
+    *)
+        ;;
+esac
```

Comment:
Note runtime installs are generally unacceptable, for testing on your PR it is fine but we must include this in our base VM images if we need it for the new test and add it as testing requirement in the podman-tests rpm.

The alternative is to not use `script` and setup the tty manually like done in 450-interactive.bats and just work with command redirection on stdout.